src/loaders/lottie/tvgLottieModifier.h

```diff
@@ -34,7 +34,11 @@ struct LottieModifier
     enum Type : uint8_t {Roundness = 0, Offset};
 
     LottieModifier* next = nullptr;
+    RenderPath* buffer;
     Type type;
+    bool chained = false;
```

Comment:
This variable looks an unnecessary complexity. doesn't mean a valid `next` chained?

Reply:
can you please rephrase this sentence: "doesn't mean a valid `next` chained?"? 
I introduced it to handle cases ```a->b->...->a```

|current chain|add|inside decorate|
|-|-|-|
|a|b|```a->chaining = true; return b->a```|
|b->a|c|```b->chaining = true; return c->b->a```|
|c->b->a|a|```a->chaining == true; return c->b->a```|
without chaining check:
|c->b->a|a|```c->chaining = true; return a->c->b->a->....``` - CYCLE|

Reply:
Got it,  Do you have any sample where the exception occurs at line 53?